### ISSUE TITLE
Accessible /new/deploy components

### ISSUE THREAD

**Issue by PaulFarault** [TOSIT-IO/tdp-ui#156]
Deploy options from the `/new/deploy` page are not accessible (cant' focus them using <kbd>Tab</kbd>).

**Comment by mdrutel** [comment]
Actually, the options from the`/new/deploy` pages ("Deploy from the DAG", "Deploy operations", "Reconfigure") are accessible (cf. image below)
![image](https://user-images.githubusercontent.com/105222830/216276518-45e820ff-6386-4061-8014-346df2cc4b63.png)

But the links are not outlined due to the attribute "focus:outline-none" ; should we remove this ?

**Comment by PaulFarault** [comment]
Yes, outlines should be displayed on focus. We may also want to add an `aria-label` as the Link is defined on a div?
